src/providers/PushProvider.ts

```diff
@@ -1,19 +1,23 @@
 import {Plugins} from '@capacitor/core';
 const {LocalNotifications} = Plugins;
 
-let someNumber = 0;
+let notificationId = 1;
```

Comment:
Wouldn't this variable reset to 1 everytime the app is reopened? Seems like that would mess with scheduling / cancelling of notifications. This likely needs to be stored and loaded from localstorage too